docs/source/developers/contributing/prerequisites.rst

```diff
@@ -106,50 +105,24 @@ If you would like to run these locally, you will need to be authenticated. The f
           
           Resource group must be created before running the test.
 
-
-If you'd like to persist resources after a test (i.e., not teardown project/resource group), you can set the environment variable ``PARAGLIDER_TEST_PERSIST`` to ``1``.
-
-**IBM** 
+IBM
+^^^
 
 #. Set environment variable ``PARAGLIDER_IBM_API_KEY`` with an ``IAM API`` key. Create a key on `IBM's web console <https://cloud.ibm.com/iam/apikeys>`_. 
 #. Set environment variable ``PARAGLIDER_IBM_RESOURCE_GROUP_ID`` with a resource group ID. 
    Pick a resource group from `IBM's web console <https://cloud.ibm.com/account/resource-groups>`__.
 
-| Cleanup function, terminating all Paraglider resources on IBM, is executed automatically when tests end, unless ``INVISINETS_TEST_PERSIST`` is set to ``1``.
-
-Editor
---------------------
-If you don't have a code editor set up for Go, we recommend VS Code. The experience with VS Code is high-quality and approachable for newcomers.
-
-Alternatively, you can choose whichever editor you are most comfortable for working on Go code. Feel free to skip this section if you want to make another choice.
-
-- `Visual Studio Code <https://code.visualstudio.com/>`_
-- `Go extension <https://marketplace.visualstudio.com/items?itemName=golang.go>`_
-
-Launching VS Code
+Persisting Resources
 ^^^^^^^^^^^^^^^^^^^^
-The best way to launch VS Code for Go is to do *File* > *Open Folder* on the repository. 
-
-You can easily do this from the command shell with ``code .``, which opens the current directory as a folder in VS Code.
 
+The functional tests will automatically clean up any resources they create before completing the test run.
+If you'd like to persist resources after a test (i.e., not teardown project/resource group), you can set the environment variable ``PARAGLIDER_TEST_PERSIST`` to ``1``.
 
-Using the Dev Container
-------------------------
-Dev Containers allow you to run a development environment using VS Code inside a container. If you want to try this:
```

Comment:
Is it still valuable to at least have the pre-reqs for the dev container? Or is it explained enough in what you link above?

Reply:
The link is the official VS Code guideline, but I think you're right in that this short statement would be helpful as dev containers are not expected knowledge. I updated the wording under "VS Code Setup".